imap_processing/mag/packet_definitions/MAG_SCI_COMBINED.xml

```diff
@@ -0,0 +1,235 @@
+			</xtce:Parameter>
+			<xtce:Parameter name="PRI_SENS" parameterTypeRef="UINT1">
+				<xtce:ShortDescription>Primary Sensor</xtce:ShortDescription>
+				<xtce:LongDescription>Primary Sensor</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SPARE1" parameterTypeRef="UINT4">
+				<xtce:ShortDescription>Spare</xtce:ShortDescription>
+				<xtce:LongDescription>Spare</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="PRI_VECSEC" parameterTypeRef="UINT3">
+				<xtce:ShortDescription>Primary Vectors per Second</xtce:ShortDescription>
+				<xtce:LongDescription>Primary Vectors per Second</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SEC_VECSEC" parameterTypeRef="UINT3">
+				<xtce:ShortDescription>Secondary Vectors per second</xtce:ShortDescription>
+				<xtce:LongDescription>Secondary Vectors per second</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SPARE2" parameterTypeRef="UINT2">
+				<xtce:ShortDescription>Spare</xtce:ShortDescription>
+				<xtce:LongDescription>Spare</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="PRI_COARSETM" parameterTypeRef="UINT32">
+				<xtce:ShortDescription>Primary Coarse Time</xtce:ShortDescription>
+				<xtce:LongDescription>Primary Coarse Time</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="PRI_FNTM" parameterTypeRef="UINT16">
+				<xtce:ShortDescription>Primary Fine Time</xtce:ShortDescription>
+				<xtce:LongDescription>Primary Fine Time</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SEC_COARSETM" parameterTypeRef="UINT32">
+				<xtce:ShortDescription>Secondary Coarse Time</xtce:ShortDescription>
+				<xtce:LongDescription>Secondary Coarse Time</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="SEC_FNTM" parameterTypeRef="UINT16">
+				<xtce:ShortDescription>Secondary Fine Time</xtce:ShortDescription>
+				<xtce:LongDescription>Secondary Fine Time</xtce:LongDescription>
+			</xtce:Parameter>
+			<xtce:Parameter name="VECTORS" parameterTypeRef="VAR_BYTE">
+				<xtce:ShortDescription>MAG Science Vectors</xtce:ShortDescription>
+			</xtce:Parameter>
+		</xtce:ParameterSet>
+		<xtce:ContainerSet>
+			<xtce:SequenceContainer name="CCSDSPacket">
+				<xtce:EntryList>
+					<xtce:ParameterRefEntry parameterRef="VERSION" />
+					<xtce:ParameterRefEntry parameterRef="TYPE" />
+					<xtce:ParameterRefEntry parameterRef="SEC_HDR_FLG" />
+					<xtce:ParameterRefEntry parameterRef="PKT_APID" />
+					<xtce:ParameterRefEntry parameterRef="SEQ_FLGS" />
+					<xtce:ParameterRefEntry parameterRef="SRC_SEQ_CTR" />
+					<xtce:ParameterRefEntry parameterRef="PKT_LEN" />
+				</xtce:EntryList>
+			</xtce:SequenceContainer>
+			<xtce:SequenceContainer name="P_MAG_SCI_NORM">
+				<xtce:BaseContainer containerRef="CCSDSPacket">
+					<xtce:RestrictionCriteria>
+						<xtce:Comparison parameterRef="PKT_APID" value="1052" useCalibratedValue="false" />
```

Comment:
FYI, If they are identical I think you can add a `comparisonList` here as well to compare against both values. Fine the way you have it though.
```python
<xtce:ComparisonList>
  <xtce:Comparison parameterRef="PKT_APID" value="1052" useCalibratedValue="false" />
  <xtce:Comparison parameterRef="PKT_APID" value="1068" useCalibratedValue="false" />
</xtce:ComparisonList>
```

Reply:
I tried that, and a few other ways, but I wasn't able to get it to work. IIRC the comparison list is only for if ALL the comparisons are true so it never evaluates to true... and there's some weird XTCE OR comparison structure that I tried, but that didn't work either.

Reply:
:+1: Not worth digging into then. Your version isn't too much extra duplication.